hbshelpers/sdkAssetUrl.js

```diff
@@ -0,0 +1,35 @@
+const RELEASE_BRANCH_REGEX = /^release\/v[0-9.]+$/;
+const HOTFIX_BRANCH_REGEX = /^hotfix\/v[0-9.]+$/;
+const SEM_VER_REGEX = /^[1-9]+$|^[1-9]+\.[0-9]+$|^[1-9]+\.[0-9]+\.[0-9]+$/;
+
+/**
+ * Given a branch (or release) of the SDK and a locale, this helper provides the correct
+ * URL in the CDN for the asset.
+ * 
+ * @param {string} branch The branch (or release) of the SDK.
+ * @param {string} locale The locale to use.
+ * @param {string} assetName The name of the desired asset.
+ * @returns {string} The CDN URL of the localized asset.
+ */
+module.exports = function sdkAssetUrl(branch, locale, assetName) {
+  const isReleasedBranch = SEM_VER_REGEX.test(branch);
+
+  let parsedBranch;
+  if (isReleasedBranch) {
+    parsedBranch = `v${branch}`;
+  } else {
+    parsedBranch = `dev/${branch.replace(/\//g, '-')}`;
+  }
+
+  const isPreReleaseBranch = 
+    RELEASE_BRANCH_REGEX.test(branch) || HOTFIX_BRANCH_REGEX.test(branch);
+  const isLocalizationSupported = 
+    (isReleasedBranch || isPreReleaseBranch) && 
+    !(locale.startsWith('en') || assetName === 'answers.css') ;
+  
+  const parsedAssetName = isLocalizationSupported ?
+    `${locale}-${assetName}` :
```

Comment:
would we ever want to just always slap the `{locale}-` prefix if the locale is not en, regardless of whether they're using a release/hotfix branch? Right now, if somebody asks for a french SDK when it doesn't exist, we'll give them an english SDK, which I think is better but curious if you had any thoughts?